web/src/demo_components/progress.dart

```diff
@@ -9,13 +9,6 @@ UiFactory<ProgressProps> Progress;
 
 @Props()
 class ProgressProps extends UiProps {
-  /// The id for the [Progress] component.
-  ///
-  /// If left unspecified one will be auto-generated for you to ensure
-  /// that the [caption] element is properly linked for accessibility purposes.
-  @override
-  String get id;
```

Comment:
@greglittlefield-wf same as what I found in the `react-dart` PR... [`@virtual` doesn't seem to make the error go away](https://github.com/cleandart/react-dart/pull/106#discussion_r85552421).